src/Extensions/Activities/Internal/ActivityLoggerHelpers.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Omex.Extensions.Activities
+{
+	internal static class ActivityLoggerHelpers
+	{
+		public static StringBuilder AppendObjStart(this StringBuilder builder) => builder.Append('{');
+
+		public static StringBuilder AppendObjEnd(this StringBuilder builder) => builder.Append('}');
+
+		public static StringBuilder AppendArrayStart(this StringBuilder builder) => builder.Append('[');
+
+		public static StringBuilder AppendArrayEnd(this StringBuilder builder) => builder.Append(']');
+
+		public static StringBuilder AppendSeparator(this StringBuilder builder) => builder.Append(',');
+
+		public static StringBuilder AppendParamName(this StringBuilder builder, string name) =>
+			builder.Append(name).Append(':');
+
+		public static StringBuilder AppendPairs<T>(this StringBuilder builder, IEnumerable<KeyValuePair<string, T>> pairs)
```

Comment:
Suggest documenting how the ToString() of T will be used in this method so that a consumer can understand how to use this correctly for complex types.

Reply:
The class could not be used outside of the project (its internal). Initially, I wanted to make it a nested class but it does not support extension methods.
I think it terms of one single project it's easier to look into implementation.

Reply:
I'd prefer it was documented (even just a note).
I find the pop-up method prompts in VS really useful for maximizing productivity when there are targeted, useful comments like this - even just as a reminder. However, I don't feel strongly on it.